core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -373,6 +373,7 @@ createOffer.fundsBox.networkFee=Mining fee
 createOffer.fundsBox.placeOfferSpinnerInfo=Offer publishing is in progress ...
 createOffer.fundsBox.paymentLabel=Bisq trade with ID {0}
 createOffer.fundsBox.fundsStructure=({0} security deposit, {1} trade fee, {2} mining fee)
+createOffer.fundsBox.fundsStructure.BSQ=({0} security deposit, {1} mining fee) + {2} trade fee
```

Comment:
Maybe this should keep the same order (deposit, trade fee, mining fee) as `createOffer.fundsBox.fundsStructure` for less confusion?

Reply:
The reason why I did it like that is, that it would otherwise look like as if it is part of the total btc funds needed.
<img width="981" alt="bildschirmfoto 2019-02-11 um 09 54 54" src="https://user-images.githubusercontent.com/170962/52553221-1edae980-2de3-11e9-8583-1d36cbff23cf.png">
So I implemented it as follows:
<img width="984" alt="bildschirmfoto 2019-02-11 um 09 56 30" src="https://user-images.githubusercontent.com/170962/52553289-534ea580-2de3-11e9-9ced-09bb2755c690.png">

Reply:
On first sight yes, but we have this order on a couple of places and it makes sense from my point of view to have it as it is right now:
<img width="604" alt="bildschirmfoto 2019-02-11 um 15 12 05" src="https://user-images.githubusercontent.com/170962/52568611-8a3ab080-2e0f-11e9-827d-b27bfbb2ad6d.png">

<img width="986" alt="bildschirmfoto 2019-02-11 um 15 11 08" src="https://user-images.githubusercontent.com/170962/52568601-83ac3900-2e0f-11e9-9948-a346eaed6086.png">

<img width="1138" alt="bildschirmfoto 2019-02-11 um 15 14 44" src="https://user-images.githubusercontent.com/170962/52568700-ce2db580-2e0f-11e9-8181-3e9bbc68de26.png">

The only problem is to mix it in the total funds field with the BSQ value.
I personally would  go with this little inconsistency and have the order in this field different between BSQ and BTC trading fees.

Reply:
Yeah, that's a lot to change, then just go with your original suggestion unless we have an obviously better way to do it.